IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml.cs

```diff
@@ -28,40 +29,27 @@ public void OnGet()
             {
                 using (var webClient = new WebClient())
                 {
-                    // grab our JSON text.
-                    string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries");
-
-                    // convert raw text to objects.
-                    List<BreweryCollection> breweryCollections = BreweryCollection.FromJson(jsonString);
-
-                    //Getting data of all Brewery
+                    string BreweriesData = string.Empty;
+                    try
+                    {
+                        BreweriesData = webClient.DownloadString("https://api.openbrewerydb.org/breweries");
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine("Error during API call - Breweries", e);
+
+                    }
+                    List<BreweryCollection> breweryCollections = BreweryCollection.FromJson(BreweriesData);
                     ViewData["BreweryCollection"] = breweryCollections;
 
-
-                    // grab our JSON text.
                     string jsonStringDetails = webClient.DownloadString("https://api.openbrewerydb.org/breweries/10-56-brewing-company-knox");
 
-                    // convert raw text to objects.
+
                     Brewery detailBrewery = Brewery.FromJson(json: jsonStringDetails);
 
-                    //Getting data of all Brewery
                     ViewData["Brewery"] = detailBrewery;
 
-                    //JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("BreweryArc.json"));
-                    //JArray Array = JArray.Parse(jsonString);
-                    //IList<string> validationEevnts = new List<string>();
-                    //if (jsonObject.IsValid(schema, out validationEevnts))
-                    //{
-                    //    var breweries = Breweries.FromJson(jsonString);
-                    //    ViewData["Breweries"] = breweries;
-                    //} 
-                    //else
-                    //{
-                    //    foreach(string evt in validationEevnts)
-                    //    {
-                    //        Console.WriteLine(evt);
-                    //    }
-                    //    ViewData["Breweries"] = new List<Breweries>();

```

Comment:
Good.

 Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.